vscodium/PKGBUILD

```diff
@@ -16,6 +16,9 @@ package() {
   install -dm755 ${pkgdir}/opt/vscodium
   bsdtar -xf VSCodium-linux-x64-${pkgver}.tar.gz -C ${pkgdir}/opt/vscodium
 
+  install -dm755 ${pkgdir}/usr/bin
+  ln -s /opt/vscodium/bin/codium ${pkgdir}/usr/bin/codium
+
```

Comment:
Maybe link it to vscodium too?

Reply:
codium is analogous to `code` (in vscode) which is to [open a file/folder from terminal](https://github.com/VSCodium/vscodium/blob/master/DOCS.md#terminal-support)

Only codium is explained in the docs